src/main/java/teho/main/Parser.java

```diff
@@ -0,0 +1,44 @@
+package teho.main;
```

Comment:
Can organize the import statements in alphabetical order. This convention makes it easier to locate specific classes and packages.